sample-configs/profile-template.yml

```diff
@@ -8,7 +8,7 @@
 #                 previous field)
 # floc: x,y Causes the field to be placed at relx,rely
 # The 'ftype' item determines the type of field used. Default is text.
-# ftype: file, text, ftext     (file search field, text input field, fixed text)
+# ftype: file, text, ftext, ttext     (file search field, text input field, fixed text)
```

Comment:
What is the new 'ttext' type?

Reply:
That's a mistake.  I had started to change the name of the field and then changed my mind

Reply:
I am going to drop this PR and submit a new one based off an earlier commit